config/database.yml

```diff
@@ -1,31 +1,33 @@
-# This file will be overwritten by configuration manager
-#   gem install mysql2 
-#
-#   Ensure the MySQL 2 gem is defined in your Gemfile
-#   gem 'mysql2'
+---
 development:
   adapter: mysql2
   database: hydranorthdev
-  username: root 
-  password:
+  username: root
+  password: 
   pool: 5
   timeout: 5000
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
   adapter: mysql2
   database: hydranorthtest
   username: root
-  password:
+  password: 
   pool: 5
   timeout: 5000
-
+staging:
+  adapter: mysql2
+  database: hydranorthstaging
+  username: hydranorth
+  host: mysql-test
```

Comment:
isn't most of this config from ansible anyways? But nice to bring over thats for sure